sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java

```diff
@@ -138,7 +138,7 @@ static void updateSetupUsingBrowser(Activity activity, String installUrl, String
         url += "&" + PARAMETER_REQUEST_ID + "=" + requestId;
         url += "&" + PARAMETER_PLATFORM + "=" + PARAMETER_PLATFORM_VALUE;
         url += "&" + PARAMETER_ENABLE_UPDATE_SETUP_FAILURE_REDIRECT_KEY + "=" + "true";
-        url += "&" + PARAMETER_INSTALL_ID + "=" + AppCenter.getInstallId().get().toString();
+        url += "&" + PARAMETER_INSTALL_ID + "=" + IdHelper.getInstallId().toString();
```

Comment:
This will crash in the work around where resume method is called after start if the SDK background thread that initialize storage is not fast enough.

That's also a reason why we should use thenAccept.

Reply:
No, it cannot be called before storage initialization.
See https://github.com/microsoft/appcenter-sdk-android/blob/9f7aff78841cf769ccef330926a48d5af5961540/sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java#L475